ansible/library/vmhost_server_info.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+from ansible.parsing.dataloader import DataLoader
+from ansible.inventory.manager import InventoryManager
+
+DOCUMENTATION = '''
+module: vmhost_server_info.py
+Ansible_version_added:  2.0.0.2
```

Comment:
This is a customized ansible module, this comment does not apply.

Reply:
`Ansible_version_added:  2.0.0.2` This comment is not necessary for a customized module.